csrc/scheduler/registry.cpp

```diff
@@ -2039,7 +2033,7 @@ class PersistentKernelScheduler : public SchedulerEntry {
     // disable the schedule if not evenly divisible on Titan RTX and
     // V100, i.e., compute architecture version 7.
     // TODO: Revisit
-    if (is_cross_grid &&
+    if (is_cross_grid && cross_grid_params.has_value() &&
```

Comment:
same here

Reply:
`Error (CLANGTIDY) [bugprone-unchecked-optional-access,-warnings-as-errors] unchecked access to optional value`